Front-End/src/pages/recommendation/Recommendation.css

```diff
@@ -0,0 +1,51 @@
+@import url("https://fonts.googleapis.com/css?family=Poppins&display=swap");
+
+.recom-container {
+    margin: 0;
+    padding: 1px;
+    width: 100%;
+}
+.recomendation {
+    font-family: "Poppins";
+    overflow-y: scroll;
+    overflow-x: hidden;
+    height: 100%;
+    justify-content: center;
+    align-items: center;
+    margin: 20px;
+}
+
+.recomendation::-webkit-scrollbar {
+    width: 2px;
+}
+
+.recomendation::-webkit-scrollbar-track {
+    box-shadow: inset 0 0 6px rgba(0, 0, 0, 0.3);
+}
+
+.recomendation::-webkit-scrollbar-thumb {
+    background-color: var(--accent);
+}
+
+.recommendation-text {
+    font-size: 15px;
+    line-height: 1.5;
+
+    margin-bottom: 20px;
+}
+
+.recommendation-text:not(:first-child) {
+    border: 1px solid #ccc;
+    padding: 5px;
+    margin: 20px;
+}
+.links {
+    /* display: flex;
```

Comment:
Just in general, no need to leave in commented code with github due to the ability to see older versions of code and just adding on extra lines of code that may not be useful (due to the fact they are not being used, it is not JS-DOC, its just commented out CSS styles).

Reply:
Agree

Reply:
Fixed.